lib/ansible/modules/clustering/kubevirt/kubevirt_cdi_upload.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+
+module: kubevirt_cdi_upload
+
+short_description: Upload local VM images to CDI Upload Proxy.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+
+description:
+  - Use Openshift Python SDK to create UploadTokenRequest objects.
+  - Transfer contents of local files to the CDI Upload Proxy.
+
+extends_documentation_fragment:
+  - k8s_auth_options
+
+options:
+  pvc_name:
+    description:
+      - Use to specify the name of the target PersistentVolumeClaim.
+    required: true
+  pvc_namespace:
+    description:
+      - Use to specify the namespace of the target PersistentVolumeClaim.
+    required: true
+  upload_host:
+    description:
+      - URL containing the host and port on which the CDI Upload Proxy is available.
+  upload_host_verify_ssl:
+    description:
+      - Whether or not to verify the CDI Upload Proxy's SSL certificates.
```

Comment:
So if we add true, system CA is used to verify, or which one?

Reply:
Yes, it seems to use standard system certs (whatever those are currently). Interestingly enough the [underlying library](http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification) also supports specifying a path to a CA bundle in addition to `true` and `false`. I wonder if in this particular case this isn't something we should also support.

Reply:
Maybe would be good to document it.

Reply:
That would force me to describe details of system–wide CA implementation on every platform supported by ansible. This module's docs are not the place for that. The general rule I've noticed is that every time there's a "verify ssl certs" on/off in a tool, the implication is always "… against whatever is the standard cert storage on your system". You only document this stuff if you allow for something more (like `ca_path`).
(This is at least true for other ansible modules, with the parameter typically called `verify_ssl`.)

Reply:
Yeah so just write, 'if not specified uses system CA ', that's good enough.you don't have to describe how to add cert to system CA store. But up to you.